docs/architecture/adr-043-interchain-identifiers.md

```diff
@@ -0,0 +1,108 @@
+- **Identifier** (the DID) of the subject with the Identifier of the Controller who is authorized to update the DID Document.
+- **Verificaton Method/s** (The cryptographic key material).
+- **Verification Relationship/s** (Describing and restricting the purposes for which the verification Methods can be used, such as Authentication).
+- **Services** related to the DID subject, with endpoints for reaching these services, such as data stores or blockchain services.
+
+For the IID Method, two additional DID Document properties have been specified, which are required for many types of on-chain assets:
+- **Linked Resources** for describing the resources which are associated with an on-chain asset, such as a media file together with its resource identifier, type, format, location and cryptographic proof (such as a hash). 
+- **Accorded Rights** for recording machine-executable capabilities, legal rights, or any other entitlements, which are accorded to the owner/controller of an asset.
+
+Linked Resources can demonstrate the distinctions between an IID and DID:
+```
+Asset IID   // self-referrant <--
+├── Linked Resources  // examples
+│   ├── Resource DID  // externally referrant -->
+│   ├── Other Resource URL
+│   ├── Another Resource CID (Content ID)
+```
+
+The **DID Syntax** for an IID conforms to the [DID-Core Syntax](), which would be expressed as follows for an asset minted from the NFT Module on a Cosmos chain:
+
+`DID:COSMOS:CHAIN:MODULE:abc123`
+
+Where: 
+- `COSMOS` is the method _(it is not the Cosmos Zone namespace)_.
+- `CHAIN` is the namespace, such as `IXO` or `REGEN` _(it is not the chain-id)_.
+- `NAMESPACE` is the asset namespace, such as `NFT` or `DOMAIN` _(as named within the context of each application chain)_.
+- `{unique identifier string} abc123` is defined by applications developers _(NFTs in two different networkscould even have the identical Inique Identifier String value and will still have completely unique IIDs)_   
+
+Unique Identifier strings need only be unique within the context of their own namespace. These may be Serial Numbers, Content Identifiers, or any another identifier method which is either based on a published standard, or is proprietary.
+
+[DID Resolvers](https://www.w3.org/TR/did-core/#:~:text=7.1%20DID%20Resolution.-,DID,-resolver) are required for each DID Method, so that users are able to successfully retrieve the DID Document for a given DID (in the W3C DID-Core standard format).
+
+It is not necessary for application chains to have a separate IID Module (or DID Module), if a specific asset module, such as `NFT`, provides the service of creating and updating IIDs and IID Document state. 
+
+An example of the extensibility of the DID syntax: a derivative demom of fungible tokens could be minted to represent the shares  IIDs could   
+
+## Decision
+
+1. Cosmos IID Method DIDs SHOULD follow the standard format:
+    -  `DID:COSMOS:` for the Cosmos Method (IID)
```

Comment:
```suggestion
    -  `DID:COSMOS:` for the Cosmos Method (IID)
```

We should discuss the use of 'cosmos' for this IID methods. I think there is an important opportunity to maintain the IID convention, where the DID only ever identifies on-chain assets and do so in a way that any Cosmos blockchain can easily adopt. This convention ensures some modest privacy benefits but is, by necessity, a different way to do things that "normal" DIDs. It *is* just a convention, but if we start promoting using IIDs to represent people, then we can't tell, for example, if a given VC with the subject as a cosmos DID/IID is about the person the DID is used to refer to, or to an on-chain asset that record data for the DID.

At the same time, there is also high demand for more traditional DIDs on Cosmos (as odd as that sounds for a tech that hasn't even been published as a formal standard). DIDs that are used to represent people or companies, are not IIDs, by definition. It may well be far more appropriate to use did:cosmos for generic identifiers and find another monicker for cosmos-based IIDs.

Once we have the software for one (either a Cosmos DID Method or a Cosmos IID Method), the software should be readily usable for the other. The main advantage of supporting a Cosmos DID Method (on which is not an IID) is that existing and emerging DID expectations are that DIDs for people use the primary authentication for the DID as a means of authenticating people. IIDs provide a way to authenticate as either the NFT or as an subset identifier defined within the IID's namespace. Those subset IDs *can* refer to people in IIDs and IIDs allow that in a way that removes the HTTPRange14 problem, but most authentication approaches I know do not actively support the notion of authenticating *as* a subidentifier.

ALTHOUGH, I just thought of a way that we might be able to get support for authenticating subidentifiers without too much of a lift. At least there might be a way to have the only real difference between IIDs and DIDs be how they are used (convention) rather than DID-level functionality like authentication and authorizations.

So, if the DIDs for people use case is important, and I think it is, we should probably use did:cosmos for that use and explore another monicker for cosmos IIDs.